packages/spacecat-shared-data-access/src/service/experiments/accessPatterns.js

```diff
@@ -96,3 +96,54 @@ export const upsertExperiment = async (
 
   return experiment;
 };
+
+/**
+ * Removes all given experiments.
+ *
+ * @param {DynamoDbClient} dynamoClient - the dynamo client.
+ * @param {DataAccessConfig} config - the data access config.
+ * @param {Logger} log - the logger.
+ * @param {ReadonlyArray<SiteCandidate>} experiments - Array of experiments to be deleted
+ * @returns {Promise<void>} - A promise that resolves when all key events are removed
+ */
+export const removeExperiments = async (
+  dynamoClient,
+  config,
+  log,
+  experiments,
+) => {
+  const tableName = config.tableNameExperiments;
```

Comment:
add a guard statement to not invoke data base operations if no or empty experiments are given